Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/Login.cs

```diff
@@ -17,7 +17,7 @@ namespace Scenes
 {
     public abstract class Login : MonoBehaviour
     {
-        public const string MainSceneName = "SampleMain";
+        public string sceneToLoad { get; private set; }
```

Comment:
@sneakzttv you've missed [field:SerializedField] attribute.

Reply:
ah, was all over the place, fixing in a sec

Reply:
done